src/main/java/com/google/graphgeckos/dashboard/storage/Builder.java

```diff
@@ -0,0 +1,47 @@
+package com.google.graphgeckos.dashboard.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+
+/**
+ * Represents the information about a single buildbot.
+ */
+@Entity(name = "builder")
+public class Builder {
+  // name of the builder
+  private final String builderName;
+  // builder logs
```

Comment:
Please capitalise the start of comments. The [style guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment) refers to Jdoc here but this should apply to regular comments too